md/hardware-and-software-requirements.md

```diff
@@ -33,7 +33,7 @@ Software required for Bonita Platform (Bonita Engine and Bonita Portal).
 | **Database**                       | (see note 2)                                     |
 | MySQL                              | 8.0.14 and higher in the 8.0.x line (see note 3) |
 | PostgreSQL                         | 11.2 and higher in the 11.x line                 |
-| SQL Server                         | 2016                                             |
+| SQL Server                         | 2017                                             |
```

Comment:
@pabloalonso Should we also have a specific focus on this update in RN page?

Reply:
it should be

Reply:
Ok, so i'll let you complete this PR, ok?

Reply:
It is in another PR
https://github.com/bonitasoft/bonita-doc/pull/1245